src/validation.cpp

```diff
@@ -47,14 +47,12 @@
 #include <util/rbf.h>
 #include <util/strencodings.h>
 #include <util/system.h>
-#include <util/time.h>
 #include <util/trace.h>
 #include <util/translation.h>
 #include <validationinterface.h>
 #include <warnings.h>
 
 #include <algorithm>
-#include <chrono>
```

Comment:
35a04e0707478f18a5c12e8979a823e119d9: Are you sure that nothing else uses those two headers?

Reply:
Ah my bad, reinstated as of a19cc42ca2